### DISCUSSION TITLE
The student pack application without an upload button.

### DISCUSSION THREAD

**Discussion by MohdSaadJames** [community/community#17736] [GitHub Education]
If you go to apply for student pack application. Where they ask for the proof. There is no upload image from files/computer option. There is only take picture option. That too with only front camera.

Please fix this issue.

I am using a Lenovo tab 4 850x with chrome.

There is no add blocker and all permissions are already granted.

**Comment by elisemoe** [comment]
Hi @MohdSaadJames  👋 sorry for the frustration! Would you be able to apply from a phone camera?

Sometimes, for a number of reasons, our system needs additional levels of verification and asks that you take an image rather than upload one. Other suggestions might be to try applying while you are on campus, if able to.

**Reply by MistyHurley** [reply]
This is inaccessible. I have an ADA-protected disability that makes it very difficult to hold still documents or a camera, and every attempt is coming out blurry. A user below writes that they are having difficulty because they are blind. This is not okay and something you need to fix.

**Reply by cgevans** [reply]
Wouldn't it make sense to place this explanation on the page requiring a camera?  The current situation is extremely confusing.

**Reply by ThalusA** [reply]
More than 30 downvotes for this suggestion, waiting for 8 months, still no change. This issue is the most upvoted one. People who don't know how to question themselves are clearly a pain in the ass in those corporate

**Reply by StephenAnimates** [reply]
What possible reason could there be for verification from an image rather than an uploaded digital document? Certainly forgery can be easier in a printed image than a digital one.

And we are all asked to risk a website gaining access to our camera or phone with no security measures highlighted? I suppose GitHub feels this is an acceptable risk for educators and students to have special account access. Not for me, thanks.

**Reply by dipee** [reply]
I am trying to take the picture from an Android device but I can only access the front camera the option to use the back camera of the device is not working. Why I this so difficult if I have a verified college email?

**Reply by Pyae24** [reply]
What about online students from other countries like me? We don't have any paper or ID to show and can only upload soft copies.

**Reply by realUjanSen** [reply]
It really sucks. I clicked on camera access yet it still doesn't display my camera view. Absolute trash.

**Reply by minhnh** [reply]
Any updates on this issue? I'm registered as a student in one university but do my research in another, so my ID is out of date and I would have to organize travel specifically to update the ID. I pay the fee every semester, however, and has the officially accepted PDF document in Germany (Semesterbescheinigung). I'm not sure why no upload button is available in my case.

**Reply by OMikkel** [reply]
Dogshit feature it stretches the uploaded images so much that the system checking the images cant see anything from the image.

I am on my desktop which has no camera. When i go to use my phone the backcamera doesnt work so i have to stand super awkward to capture a photo of my screen. When i finally capture an image it is super blurry and stretched.

**Reply by djangid63** [reply]
It's more like Github intesionaly don't want us to get the pack for free and want us to pay for it

**Reply by yoloytz** [reply]
> Dogshit feature it stretches the uploaded images so much that the system checking the images cant see anything from the image.狗屎功能 它会将上传的图像拉伸得如此之多，以至于检查图像的系统无法从图像中看到任何内容。
> 
> I am on my desktop which has no camera. When i go to use my phone the backcamera doesnt work so i have to stand super awkward to capture a photo of my screen. When i finally capture an image it is super blurry and stretched.我在没有摄像头的桌面上。当我去使用手机时，后置摄像头不起作用，所以我不得不非常尴尬地站着拍摄屏幕照片。当我最终捕捉到一张图像时，它非常模糊和拉伸。

yes i got same problem. must take photos by standing...and the photo is so blurry

**Reply by wjabrac** [reply]
Oh sure I'll just hop over the ocean in JULY for you github, would you like me to stop by and pick up the mona lisa and give you sloppy toppy on my way?

**Reply by LPbigFish** [reply]
Please fix your OCR, I am taking a photo of clear "Confirmation of studies" from my phone and it would just point me back to the Upload proof page with no message

**Reply by Faqih01-a** [reply]
dude its been what 3 years yeah? AND YOU DID NOT FIX THE ISSUE
DUDE I ONLY HAVE THE FILE FOR MY STUDENT ID NOT PHYSICALLY, I AIN'T GONNA WAIT FOR ANOTHER YEAR JUST TO GET MY ID!?
FOR THE LOVE OF GOD PLS DON'T WASTE MY TIME YOU SON OF AN IDIOT

**Reply by SPOFETA** [reply]
I've been trying to take the goddamn photo but it comes out blurry all the times, I even managed to make the document stay still on the table and the phone still by making a tower of books and even without any movement the photo comes out all blurry and illegible. What am I supposed to do???

**Reply by dragao1995** [reply]
Tutorial
1. Open the browser on the computer.
2. You might have to enter the university's coordinates (latitude and longitude) into the browser.
3. Select or register the university.
4. Choose option 1, Student ID Card (or Student Card).
5. Take a picture of the student ID card (with your photo) using the webcam.

**Comment by aidenh37** [comment]
Having the same issue. Require GitHub for my university program. No upload button on _any_ browser or device.

No VPN. No AdBlock. I do keep getting pulled up for captcha on my IP for some reason across the web, despite no robots and completely malware-free.

None of my webcams can focus on my student ID well enough to be read. Website won't use camera on iOS, despite having permissions. Literally have no way of verification if there's no manual method here. What a stupid system.

Need urgent assistance for this issue.

**Reply by Erogunaiye** [reply]
I've uploaded mine but I'm yet to get the student pack

**Reply by RealYusufIsmail** [reply]
Be patient.

**Reply by jon-peel** [reply]
I also can't get the camera to work on iOS.
My laptop has no webcam.
The site has no upload button.

**Reply by AbdullahAjayi** [reply]
Hello, have you gotten yours?

**Comment by RealYusufIsmail** [comment]
Try to use a different device. I had the same issue with some thing similar and using a different device and it worked.

**Reply by MohdSaadJames** [reply]
Can you please share the screenshot?

**Reply by RealYusufIsmail** [reply]
I was trying to login into the eclipse website but it did not show the verification button but on my phone it did. So i assume it is the same issue. I can not share a screen shot as that computer is borken.

**Reply by EddieMusick** [reply]
I've tried three different devices, I will try a forth. This is a f***ing ridiculously broken process.

**Reply by EddieMusick** [reply]
Welp, I tried all the devices I own. Submitted a help ticket, not expecting to hear back in time for me to use this for my project.

**Reply by softwaredevelop** [reply]
This is not a Student Developer Pack this a Super Secret Project.

**Comment by webik150** [comment]
Shame this is still an issue. I've tried two different computers and nothing worked. In the end I used my phone to take the picture, but the camera ONLY works for a single frame, when changing between the front and the back camera. 

After that I got rejected because apparently my school doesn't have a campus in my country even though I daily go there. 🤡

**Reply by harrystaley** [reply]
This is just simply inaccessible. I have to hold up a copy of my schedule for the camera. I am blind and do not know where the camera is on my laptop. Please fix this NOW. Also how do you expect for the camera to give a good image with someones fingers in the way?

**Comment by shamnad-sherief** [comment]
You can try:-
- Use your system camera and face your head and ID proof. This will help you to get the application process.
- You might see the upload button when you add your face with ID proof. Then you can retake it and upload through the files. 
- You can use applications like droidcamera to add a clear picture.

**Reply by lishylishylishy** [reply]
the camera couldnt make a clear capture, then github refused my application because of the poor quality of the pic,  it doesnt make sense

**Comment by Seqaeon** [comment]
Still unable to verify, can back camera support be enabled.

**Comment by cgevans** [comment]
While rather ridiculous, you can use a virtual camera device (eg, through OBS Studio or v4l2loopback) to add a document.

In my case, applying to renew my teacher benefits, my ID does not have a date on it, and as I try to be as paperless is feasible for me, all affiliation documents I have with dates are PDFs or online.  It seems unusual to expect someone to print out a document, then take a photograph of it with the camera of the same laptop that had the document on it in the first place.  I added a document with OBS and my renewal was automatically approved.  

It's also not clear how a camera image is better for verification than the original PDF.

**Reply by EltonSHIN** [reply]
it worked!!!!!! pretty a good way!!!! everyone should know it.

**Reply by EltonSHIN** [reply]
please note: firefox supports virtual camera by default, while chrome and edge does not.

**Reply by james-togbah** [reply]
Could you navigate how you went about uploading via OBS to capture my student profile? GitHub is only giving me option to take a picture with my webcam. Whenever I open OBS studio the camera on GitHub does not work.
Thanks in advance.
![Screenshot 2024-07-01 115601](https://github.com/community/community/assets/174275362/aae2bdb6-8ffa-4a71-a95a-954fa2441c23)

**Reply by AykutUfuk** [reply]
can you please tell us more clearly how you did it ?

**Comment by elvispy** [comment]
Github, please bear in mind that universities with little economical resources don't issue a student ID. Me, being a paraguayan citizen and having studied for more than 5 years in Brazil, I can tell you that it is not customary to have a student ID. That is, apart from having to take a photo with your computer, you have to put extra effort to focus your camera on a printed document such as a transcript with fontsize 12. This is nonsense and a huge discourage for students from all around the world to have the student developer pack.

**Comment by Codaqua** [comment]
Good morning,

I would like to sign up for the GitHub student programme. I am currently a student of a degree in a university that works exclusively in distance mode. 
When I try to register, the form gives me an error because my location is far from the university, which is normal. And I don't have any document that says that I am studying in distance mode because it is the only mode that my university has.
I look forward to your comments.

Thank you.

**Comment by iumyidol** [comment]
Cant believe this is still an issue today. Come on, fix the camera bug, or give us the upload button !!

**Reply by ZhangzihanGit** [reply]
I'm sure this is not a bug, they did it intentionally.

**Reply by djanym** [reply]
For sure it's not a bug. But it's a very confusing thing. Different universities have different formats for proofing documents. And not always you can use a web camera to capture it.
Github: if you're a student you can obtain a student pack.
50% of students: we can't verify our student status with a webcam. But we could upload a proof doc.
Github whistling...

**Reply by softwaredevelop** [reply]
It's entirely up to luck. I tried to take a photo with my mobile phone X times, and after X+1 attempts, the upload button appeared.

**Comment by swap72** [comment]
> If you go to apply for student pack application. Where they ask for the proof. There is no upload image from files/computer option. There is only take picture option. That too with only front camera.
> 
> Please fix this issue.
> 
> I am using a Lenovo tab 4 850x with chrome.
> 
> There is no add blocker and all permissions are already granted.

simply drag and drop it works but it thows some other error, but try it anyways.

**Comment by JittoJoseph** [comment]
after i upload a photo id proof, and click process my application, it dosen't go to any other page, looks like it simply reloads the id proof page, i'm stuck help out

**Comment by dragao1995** [comment]
The problem here is that my college is distance education (EAD), so I can't go to the campus, and the upload button has also disappeared, and when I try to send the ID, it gets denied.

**Reply by lishylishylishy** [reply]
i have the same problem! 
i couldnt update files matching quanlity requirements

**Comment by dom976** [comment]
I cannot upload a file too, and when I use the phone I can only use the front camera

**Comment by alfaruqii** [comment]
2024 and they still have this issue

**Comment by IZ-ZI** [comment]
For people who are having trouble uploading the photo, one possible reason is that the file size is too large, in which case the server returns a code 413 payload too large and refreshes the page without any warning. Try compressing the photo. You may still get the error if the photo is just slightly under 1MB; It needs to be well under that limit.

**Comment by Astaranyx** [comment]
Same issue - no way to upload proof of studentness; University does not issue student ID cards - nothing to take a 'pic' of. This situation is EXTREMELY frustrating.

**Reply by samus-aran** [reply]
Hi @Astaranyx you are able to supply other types of evidence. 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [What types of evidence does the Student Developer Pack application accept?](https://github.com/orgs/community/discussions/111352#what-types-of-evidence-does-the-student-developer-pack-application-accept)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion/reply and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by astro-jax** [reply]
@samus-aran How do you upload these other documents ? The link for the FAQ suggests that you can include documents such as a transcript or enrollment letter, but you are not able to upload a PDF so I am unable to move past this step. What is the workaround to be able to submit these requested documents ??

**Reply by james-togbah** [reply]
Its ghetto but I have two computers. I was able to use one if my laptop to cast my documents to a tv screen then I used my other laptop front facing camera or webcam to capture the picture of the tv screen. Took me a few try but finally got it and eventually got approved.

**Reply by nbhav** [reply]
@james-togbah  thats hilarious, github 101% wants to make it difficult to get the student pack. Not sure why all it does is make people want to use gitlab or bitbucket instead.

**Comment by evazheng8** [comment]
can't believe this is still broken lol

**Comment by pongelon** [comment]
this is an atrocious process just to upload a simple image. github get it together, its a disaster that JetBrains chose to depend on you

**Comment by yzanhua** [comment]
Still having this issue. Not able to use the back camera and it does not recognize the date on my student ID

**Comment by Stdaniel2** [comment]
> If you go to apply for student pack application. Where they ask for the proof. There is no upload image from files/computer option. There is only take picture option. That too with only front camera.
> 
> Please fix this issue.
> 
> I am using a Lenovo tab 4 850x with chrome.
> 
> There is no add blocker and all permissions are already granted.

I'm having the same issue. I'm very frustrated right now.

**Comment by mirzaabdullaht** [comment]
<img width="215" alt="image" src="https://github.com/user-attachments/assets/41bd589b-f9aa-41a0-92aa-fab2cd4548ca">

Hey everyone,
I wanted to share my experience with getting the GitHub Student Developer Pack. I ran into the same problem many of you are facing - there was no option to upload documents, and when I tried using the live camera, my application kept getting rejected.

Here's what I did:

I tried applying from different devices
I emailed GitHub's customer support
I even used a MacBook because its camera is way better than other laptops
My student ID was a bit tricky because it had two pages - one with my name and the other with the expiry date. After a bunch of rejections and following their suggestions, I gave it another shot with the MacBook. It took 5 rejections over a few days, but on my 6th try, it finally worked!

Now I'm just waiting for the benefits to kick in, which should take about 3 days.

Some tips from my experience:

Don't give up. Keep trying until you get approved.
Make sure to apply before you graduate. I only had 2 months left on my student ID.
Try to get the clearest picture possible of your ID.
I know how frustrating it is when you're trying to upload a clear document but can't, and you keep getting rejected. But hang in there!

My GitHub education request finally got approved.

Good luck to all of you trying to get your GitHub Education Pack :)

**Reply by m0zee** [reply]
How did you upload both the front and back at once? It only allows one picture for the request. I have the same problem: one side has the name, and the other side has the expiry date.

**Reply by mirzaabdullaht** [reply]
For Few attempts i captured front side, and few attempts i captured back side, and luckily after 5 rejections, then my application was approved.

**Comment by roa9618** [comment]
If you're a MacBook and iPhone user and it's hard to upload images for the above reasons, I'm sharing my own tips.

It has the ability to use the iPhone as a webcam for your MacBook.

After accessing the web on the MacBook, you can take photos using the iPhone by selecting the iPhone from the camera selection.

I took a picture and uploaded the certificate of enrollment using the iPhone and it was approved.

**Comment by AntonioFranciulli** [comment]
This is still an ongoing issue that, from what one can see in the comments, its still affecting thousands of students globally. The webcam approach is far from ideal, specially for colleges that don't provide physical IDs for students and the only means of validating ones enrolment are PDFs signed by the authorities.

**Comment by Chris-Welton** [comment]
Not working for me either. I took a pic of the document with my front (and only) camera to my phone, but got rejected because the image was not clear enough (duh). Completely Ridiculous.

**Comment by dayanfcosta** [comment]
This feature is completely ridiculous. I set a tripod with my phone, during the live camera everything was pretty clear, but they so some processing after shooting the pic and before uploading it that makes it completely unclear. I take the picture and them it's completely blurred. That's a shame of feature.

**Comment by AlCapone1234** [comment]
I cry everytime I use this shit website

**Comment by Tom60chat** [comment]
Hello everyone,
I found a way to restore the upload button !
https://www.youtube-nocookie.com/embed/meuzrVX7CM0?si=_divwW8h8-etta3x

**Reply by FlaviusBurghilaOPIT** [reply]
+1, only this one worked.
Thank you

**Comment by jessic-campos** [comment]
A tip that worked for me was using my cell phone to take a photo of my document that was open on my notebook, this way it maintained a good resolution, but the camera tends to have problems switching from the front to the rear, I pressed the rotation button a few times, but it worked.

**Comment by ZainJaved869** [comment]
I will face the same issue in GitHub student developer pack have only the start camera option and have no upload button option.what I need to do .
